apps/frontend/src/utils/constants.ts

```diff
@@ -290,7 +564,9 @@ export const INITIATE_FIELDS = {
 				b2c: ["001"],
 				retail: ["001"],
 				subscription: ["001"],
-				logistics: ["TAT Breach, 007"], // Follow this format if new options are added.
+				logistics: ["TAT Breach, 007"],
```

Comment:
Please update this section of code. It seems this overrides the current/latest settings. Please make sure to not overwrite any latest changes which can affect others' work.